lib/typeDefs/book.ts

```diff
@@ -0,0 +1,25 @@
+export const bookType = `
+type Book {
```

Comment:
`implements Node` Can read up on Node implementation, thats what allows to have global unique objects - https://graphql.org/learn/global-object-identification/#node-interface